py/server/deephaven/agg.py

```diff
@@ -269,17 +278,29 @@ def std(cols: Union[str, List[str]] = None) -> Aggregation:
     return Aggregation(j_agg_spec=_JAggSpec.std(), cols=cols)
 
 
-def unique(cols: Union[str, List[str]] = None) -> Aggregation:
-    """Create a Unique aggregation.
+def unique(cols: Union[str, List[str]] = None, include_nulls: bool = False, null_sentinel: Any = None) -> Aggregation:
```

Comment:
This work should cover similar args to `count_distinct`. Also, is the wrapper missing for distinct?